xml/schema/Price/PriceField.xml

```diff
@@ -24,6 +24,11 @@
     <type>int unsigned</type>
     <required>true</required>
     <comment>FK to civicrm_price_set</comment>
+    <pseudoconstant>
+      <table>civicrm_price_set</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>label</labelColumn>
```

Comment:
@pradpnayak i think this needs to be title not label

Reply:
Yes, you are right. Have updated the line.